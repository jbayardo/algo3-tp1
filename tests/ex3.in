a ;b c;c b
a c;b c;c ab
a ;b ;c d;d c
a b;b da;c ;d b
a d;b c;c bd;d ca
a ;b ;c d;d c;e 
a ;b ed;c ;d b;e b
a edb;b a;c ;d a;e a
a d;b d;c d;d ebca;e d
a ;b c;c b;d ;e ;f 
a ;b ;c d;d fc;e ;f d
a ef;b ;c f;d ;e a;f ca
a ;b ed;c e;d b;e bfc;f e
a c;b c;c aebd;d c;e fc;f e
a e;b ;c ;d ;e a;f ;g 
a ;b c;c b;d ;e f;f e;g 
a g;b e;c f;d ;e b;f c;g a
a d;b ;c ;d gae;e fd;f e;g d
a bf;b ac;c b;d f;e ;f dag;g f
a bd;b fa;c ;d fa;e f;f gbde;g f
a f;b ;c ;d ;e ;f a;g ;h 
a ;b ;c ;d ;e ;f g;g fh;h g
a ;b ;c d;d c;e ;f g;g hf;h g
a gd;b ;c ;d a;e g;f g;g fea;h 
a he;b ge;c ;d h;e ba;f ;g b;h ad
a df;b e;c e;d a;e hbc;f ha;g ;h fe
a b;b ahe;c e;d e;e cdbg;f g;g fe;h b
a ;b ;c ;d ;e h;f ;g ;h e;i 
a b;b a;c ;d ;e ;f ;g h;h g;i 
a ;b ;c dh;d cf;e ;f d;g ;h c;i 
a ;b d;c ;d fb;e i;f d;g i;h ;i ge
a g;b h;c g;d ;e ;f hi;g ca;h fb;i f
a i;b ei;c id;d c;e b;f ;g h;h g;i acb
a c;b dg;c iha;d b;e g;f ;g eib;h c;i cg
a ce;b hdf;c ae;d b;e gca;f hb;g e;h bf;i 
a ;b ;c ;d ;e ;f ;g ;h ;i j;j i
a ;b ;c ;d ;e ;f ;g j;h i;i h;j g
a ;b ;c g;d i;e ;f ;g c;h j;i d;j h
a ;b e;c j;d ;e gb;f ;g e;h i;i h;j c
a h;b g;c ;d h;e f;f ei;g b;h da;i f;j 
a e;b i;c ;d fi;e a;f d;g h;h jg;i bd;j h
a df;b cgj;c bgf;d a;e ;f ca;g bc;h ;i ;j b
a i;b fc;c gdb;d c;e i;f b;g ic;h ;i geaj;j i
a i;b ej;c i;d j;e bifg;f e;g ej;h ;i ace;j dbg
a ;b ;c ;d ;e ;f ;g ;h k;i ;j ;k h
a ;b ;c i;d ;e ;f ;g ;h ;i jc;j i;k 
a ;b ;c ;d gf;e i;f d;g d;h ;i e;j ;k 
a ;b ;c ;d i;e j;f ;g i;h ;i gd;j ke;k j
a ;b d;c e;d ibfe;e dc;f d;g ;h ;i d;j ;k 
a b;b a;c e;d e;e cd;f j;g k;h i;i h;j f;k g
a hed;b ;c d;d ca;e gka;f ;g e;h aj;i ;j h;k e
a hd;b c;c b;d gaj;e ;f ig;g df;h aj;i f;j hd;k 
a c;b ;c gaei;d ;e icj;f g;g fck;h ;i eck;j e;k gi
a ;b f;c ejh;d fe;e cdf;f dhbe;g ;h kfci;i h;j c;k h