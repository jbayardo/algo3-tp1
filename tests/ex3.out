1 abc
1 abc
1 abcd
1 abdc
1 abcd
1 abcde
1 acdbe
2 abcde
2 abcde
1 abcdef
1 abcdfe
1 aebdcf
2 acefbd
2 abcdef
1 abcdfge
1 abcdefg
1 abecfdg
2 abcfedg
2 abcedfg
3 abcefgd
1 abcdeghf
1 abcdefgh
1 abcdefgh
2 adbchefg
1 aebgcfdh
2 adgbcehf
2 abcedgfh
1 abcdehfgi
1 abcdefghi
1 abefdchgi
1 abdfceigh
1 abhfidecg
2 abefghdci
2 achfdbegi
2 acdbfhige
1 abcdefghij
1 abcdefgjhi
1 abcgdiefhj
1 abegcjdfhi
1 abgcefijdh
1 abidfcghje
2 adehijbgcf
2 aehfbcdgji
2 achdjbgefi
1 abcdefghkij
1 abcijdefghk
1 abceifdghjk
1 abcdigejkfh
2 abfdiecghjk
1 abcedfjgkhi
4 adcbfigekjh
3 abcekifgdjh
3 abdhfgkicej
4 abdfecjhikg
